frontend/src/components/common/form/ParentSelect.tsx

```diff
@@ -31,6 +31,8 @@ interface IParentSelect {
   setClearSelected?: Function;
   /** Event occurs when the selection changes. */
   onChange?: (vals: any) => void;
+  /** get the component to select the item with closest label match to the input provided */
+  selectClosest?: boolean;
```

Comment:
Does this default to `true`?  I'd think this would almost always be the desired behaviour.